### ISSUE TITLE
Please update the information on my Glyph: MemeSage

### ISSUE THREAD

**Issue by MSageVR** [pegasy/glyphs.wf#34]
Please update the information on my Glyph: MemeSage

Hello, I didn't know where else to send this, but the information under my glyph is outdated by about 3 years or so. I am no longer a warframe partner, I no longer make warframe content, and there is currently no way to obtain my glyph in-game. My Discord is no longer a warframe discord, and it's the discord I use for my work and my current community, and it's annoying when people keep joining and leaving when they realize the discord is no longer a warframe discord. 

Long story short, the glyph can no longer be obtained in any way shape or form, unless I just personally feel like giving you a code.

Best way to contact me is through my email address: info@MSage.co

**Comment by zopney** [comment]
Removed acquisition methods as requested. No verification is needed. Please allow up to 24 hours for changes to be reflected.

Please submit a new request if further changes are required.
